src/protos.ts

```diff
@@ -0,0 +1,29 @@
+export interface TreeProto {
```

Comment:
Should this be `.d.ts`?

Reply:
> Would still consider putting all type decls into a bento-compiler namespace so code can import {TreeProto} from 'bento-compiler'

They already don't need to include paths, as written in the description. 
The `index.ts` file exports what we want to expose.

Reply:
I see the confusion, I wasn't very clear above. I was referring to within the project, not just for consumers.